src/api/index.js

```diff
@@ -708,6 +710,10 @@ class API {
     )
     return validateRequestAsJSON(response)
   }
+
+  _typescriptIntegrationTest(input) {
+    return thisIsATest(input)
+  }
```

Comment:
i added this fake code to exemplify how to add new ts code into existing js. felt alright to commit it because the `api/` directory is an internal implementation to the exported `Client` & `Account` classes.